lib/payloadAction.ts

```diff
@@ -74,6 +74,17 @@ export const  handlePayloadActions = async (app: IApp, read: IRead,  modify: IMo
 
                         // Start blackout window
                         if (params.continue_blackout) {
+                            const continueBlackoutTask = {
+                                id: JobName.QUEUE_WINDOW_SCHEDULER,
+                                when: `120 seconds`,
```

Comment:
In the past we faced some issue with passing literal strings here. It was scheduling job with some invalid date. And using Date object works fine https://github.com/WideChat/Apps.Dialogflow/blob/master/lib/payloadAction.ts#L63 Can you please try it?